test/project-service.ts

```diff
@@ -39,21 +39,20 @@ class ProjectIntegrationTest {
 		return projectService.createProject(projectName);
 	}
 
-	public getDefaultTemplatePath(): IFuture<string> {
+	public getDefaultTemplatePath(templateName: string): IFuture<string> {
```

Comment:
This function should be renamed for example `getNpmPackagePath(packageName: string)`